database/db.js

```diff
@@ -1,58 +1,116 @@
 const Student = require("../models/Student");
-
+const fs = require("fs");
+const application_db = require("../database/application_db")
+if ( fs.existsSync("config/importantPng.png")){
 module.exports = {
 	addStudent: async function(studentObj) {
-    let format = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-    if(studentObj.email.toLowerCase().match(format) && studentObj.guardianEmail.toLowerCase().match(format)) {
-      const newStudent = new Student({
-        first_name: studentObj.first_name,
-        last_name: studentObj.last_name,
-        grade: studentObj.grade,
-        school: studentObj.school,
-        email: studentObj.email,
-        phone_number: studentObj.phone_number,
-        dateOfBirth: studentObj.dateOfBirth,
-        guardianPhone: studentObj.guardianPhone,
-        guardianEmail: studentObj.guardianEmail,
-        notes: studentObj.notes,
-        interestsAndHobies: studentObj.interestsAndHobies,
-        id_number: `${studentObj.last_name}.${ await module.exports.getLastNameCount(studentObj.last_name)}`,
-        program_list: studentObj.program_list,
-        status: "active",
+    if (validateStudent(studentObj)) {
+        guardianPhoneDeformated = studentObj.guardianPhone.replaceAll('(',"");
```

Comment:
I would recommend creating a function to do this a little more dynamically... something like this (pseudocode)

```js
replace_chars = ["(", ")", "-"];
foreach (char in replace_chars) {
  phoneString.replaceAll(char, "");
}
```